src/eddymotion/model/base.py

```diff
@@ -241,43 +275,41 @@ def __init__(self, **kwargs):
 
         Parameters
         ----------
-        gtab : :obj:`~numpy.ndarray`
-            An :math:`N \times 4` table, where rows (*N*) are diffusion gradients and
-            columns are b-vector components and corresponding b-value, respectively.
-        th_low : :obj:`~numbers.Number`
+        th_low : :obj:`numbers.Number`
             A lower bound for the b-value corresponding to the diffusion weighted images
             that will be averaged.
-        th_high : :obj:`~numbers.Number`
+        th_high : :obj:`numbers.Number`
             An upper bound for the b-value corresponding to the diffusion weighted images
             that will be averaged.
         bias : :obj:`bool`
             Whether the overall distribution of each diffusion weighted image will be
-            standardized and centered around the global 75th percentile.
+            standardized and centered around the global ``percentil_percentage`` percentile.
         stat : :obj:`str`
             Whether the summary statistic to apply is ``"mean"`` or ``"median"``.
 
         """
-        self._th_low = kwargs.get("th_low", 50)
-        self._th_high = kwargs.get("th_high", 10000)
+        super().__init__(**kwargs)
+
+        self._th_low = kwargs.get("th_low", bval_lower_th)
+        self._th_high = kwargs.get("th_high", bval_upper_th)
         self._bias = kwargs.get("bias", True)
         self._stat = kwargs.get("stat", "median")
         self._data = None
 
     def fit(self, data, **kwargs):
         """Calculate the average."""
-        gtab = kwargs.pop("gtab", None)
         # Select the interval of b-values for which DWIs will be averaged
         b_mask = (
-            ((gtab[3] >= self._th_low) & (gtab[3] <= self._th_high))
-            if gtab is not None
+            ((self._gtab[3] >= self._th_low) & (self._gtab[3] <= self._th_high))
+            if self._gtab is not None
             else np.ones((data.shape[-1],), dtype=bool)
```

Comment:
Unless we model DSI acquisitions `self._gtab` will not be none, so this `if/else` block is not necessary to me.

Reply:
Here we do want to use the input gtab, as opposed to the global gtab, which potentially contains the left-out gradient.